src/consolelogger.c

```diff
@@ -9,26 +9,46 @@
 
 #if (defined(_MSC_VER)) && (!(defined WINCE))
 #include "windows.h"
+#include "tchar.h"
+
+
+wchar_t *  to_wchar(const char * strA)
+{
+    wchar_t*  resultW;
+    if (strA == NULL)
+    {
+        return NULL; 
+    }
+    DWORD dwNum = MultiByteToWideChar(CP_ACP, 0, strA, -1, NULL, 0);
```

Comment:
>CP_ACP [](start = 38, length = 6)

CP_ACP: This value can be different on different computers, even on the same network. It can be changed on the same computer, leading to stored data becoming irrecoverably corrupted. This value is only intended for temporary use and permanent storage should use UTF-16 or UTF-8 if possible.

from: [https://docs.microsoft.com/en-us/windows/desktop/api/stringapiset/nf-stringapiset-multibytetowidechar](https://docs.microsoft.com/en-us/windows/desktop/api/stringapiset/nf-stringapiset-multibytetowidechar)

to be safe?

Reply:
changed to CP_UTF8, which is UTF-8.